### DISCUSSION TITLE
C:1:Electricity_Consumed_Total stucked to 0

### DISCUSSION THREAD

**Discussion by corruptux** [Arnold-n/P1P2MQTT#65] [Q&A]
Hello!

With this firmwares (P1P2-bridge-esp8266 v0.9.32 P1P2Monitor-v0.9.32) the topic C:1:Electricity_Consumed_Total is set to 0 and is not updating. 

Others, like Electricity_Consumed_Compressor_Heating, Electricity_Consumed_Backup_Heating and Electricity_Consumed_Compressor_DHW, are updating with C2 and with the Control Panel menus.

Electricity_Consumed_Compressor_Cooling is also remain 0.
I see in an [message ](https://github.com/Arnold-n/P1P2Serial/discussions/36#discussioncomment-4184398) from Arnold, the topic were working fine...

Can I do something to fix that?
Thx

**Comment by Arnold-n** [comment]
Hi @corruptux,

Strange; what is the value of total electricity consumption on the wall room main controller and in P1P2/+/C/2? In some Daikin systems the counters seem to stop working even on the wall room controller, though in my experience they remain stuck and are not reset to 0: in my system Electricity_Consumed_Compressor_Heating remains stuck at 16777 since 2018, far before I started my P1/P2 activity, while Electricity_Consumed_Total continues to increase here (also with 0.9.32) - very weird. I did play with an external electricity meter which perhaps corrupted the Daikin system settings.

Could you please share the  raw 4000B80x* messages, especially the 4000B800* message, in P1P2/R/#, and the counter value P1P2/+/C/1/Electricity_Consumed_Total and the corresponding value on the wall room controller ?

Thanks!

**Comment by corruptux** [comment]
Hello,

There is not information regarding Electricity consumed in the Main Controller. Only Energy Produced. Now I can confirm that none of the values related to electricity are changing. They remain as they were when I connected your device. It may be a problem with the settings of my Main Controller. I don’t know when those values were generated.

This branch P1P2/+/C/2 in MQTT does not exist.
 
My P1P2MQTT is still in monitoring mode.
Thank you

**Comment by Arnold-n** [comment]
So it sounds like your Daikin system unfortunately does not measure electricity consumed, so the counters remain 0. P1P2/+/C/2/# will only become available if "C2" mode is active but the values will remain 0 too in your system.
